packages/lesmis/src/components/Breadcrumbs.js

```diff
@@ -0,0 +1,81 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ *
+ */
+import React from 'react';
+import styled from 'styled-components';
+import { Link as RouterLink } from 'react-router-dom';
+import MuiBreadcrumbs from '@material-ui/core/Breadcrumbs';
+import MuiLink from '@material-ui/core/Link';
+import NavigateNextIcon from '@material-ui/icons/NavigateNext';
+import { useCurrentOrgUnitData } from '../api';
+
+const StyledBreadcrumbs = styled(MuiBreadcrumbs)`
+  font-size: 0.75rem;
+  line-height: 0.875rem;
+  text-transform: capitalize;
+
+  .MuiBreadcrumbs-separator {
+    margin-left: 0.2rem;
+    margin-right: 0.3rem;
+
+    svg {
+      font-size: 0.75rem;
+    }
+  }
+`;
+
+const ActiveSegment = styled.span`
+  color: ${props => props.theme.palette.primary.main};
+`;
+
+const Link = props => <MuiLink color="inherit" {...props} component={RouterLink} />;
+
+const getSegments = (orgUnits, orgUnitCode, heirarchy = []) => {
+  const orgUnit = orgUnits.find(entity => entity.organisationUnitCode === orgUnitCode);
+
+  if (!orgUnit) {
+    return [];
+  }
+
+  const newHeirarchy = [...heirarchy, orgUnit];
+
+  if (orgUnit.type === 'Country') {
+    return newHeirarchy.reverse();
+  }
+  return getSegments(orgUnits, orgUnit.parent, newHeirarchy);
+};
+
+const useBreadcrumbs = () => {
+  const query = useCurrentOrgUnitData({ includeCountryData: true });
+
+  if (!query.data) {
+    return { ...query, segments: [] };
+  }
+
+  const { countryHierarchy, organisationUnitCode } = query.data;
+  const segments = getSegments(countryHierarchy, organisationUnitCode);
+
+  return { ...query, segments };
+};
+
+export const Breadcrumbs = () => {
```

Comment:
Since the ui component is basically just an implementation of material-ui and this component is more about mapping data to the url, I decided that it's better to make this a component in LESMIS and not a re-usable component in the ui-components package.